resources/pretty.js

```diff
@@ -14,7 +14,7 @@ const INVERSE = '\x1b[7m';
 const RESET = '\x1b[0m';
 const YELLOW = '\x1b[33m';
 
-const options = ['--trailing-comma=es5'];
+const options = ['--config ../../.prettierrc'];
```

Comment:
These options should be separate (for `spawn`), and we should pass a fixed path so that it doesn't matter where the script is called from.

```suggestion
const options = ['--config', `${__dirname}/../.prettierrc`];
```

Reply:
really, the script is so simple now, we could just invoke it directly